CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
 - Migrate Dagger from KAPT to KSP
 - Bump AndroidX Preferences to v1.2.1
 
+### Changes
+
+- Enable `Clinical Decision Support` for all facilities in SriLanka.
```

Comment:
We can remove the code blocks around `Clinical Decision Support` as it's not a code block.

Reply:
@msasikanth Done.